preload

```diff
@@ -285,7 +285,7 @@ $prefix/$app
     for app in $APPS
     do
       if [ -f "${DIRECTORY}/data/status/${app}" ];then
-        echo "${app} ($(< "${DIRECTORY}/data/status/${app}" 2>/dev/null));${DIRECTORY}/apps/${app}/icon-64.png;:exec "\""echo '$prefix/${app}'"\"""
+        echo "${app} ($(while read line; do echo $line; done 2>/dev/null < "${DIRECTORY}/data/status/${app}" 2>/dev/null));${DIRECTORY}/apps/${app}/icon-64.png;:exec "\""echo '$prefix/${app}'"\"""
```

Comment:
Is there a reason to read all lines from the app's status file when it's always one line anyway?
For cases when you do want to read an entire file, that still wouldn't need a while loop.
```
read -d '\0'
```

Reply:
I get that you don't need a while loop just to read one line. the problem is it is the most efficient method.

this is all part of the speedup, when you go from something that takes 0.002 second to something literally less than 0.000 seconds, and do this over 500 times (each app category and all apps in that category run this), that turns into over a second of timesave